angular.json

```diff
@@ -25,7 +25,12 @@
             "tsConfig": "tsconfig.json",
             "assets": [
               "src/favicon.ico",
-              "src/assets"
+              "src/assets",
+              {
+                "glob": "**/*",
+                "input": "./node_modules/midi/examples/soundfont/",
+                "output": "./midi/examples/soundfont/"
```

Comment:
We have a hosted folder called assets (I am not sure if it is exactly called assets). Try copying these files into that folder. Then they can be accessed using a url. Example link: https://www.oppia.org/assets/images/splash/dsk_community_background.png

Reply:
as discussed, lets leae this as it is